mappings/net/minecraft/client/gui/widget/WidgetRealmsButton.mapping

```diff
@@ -0,0 +1,41 @@
+CLASS none/bfj net/minecraft/client/gui/widget/WidgetRealmsButton
+	FIELD o realmsButton Lnet/minecraft/realms/RealmsButton;
+	METHOD <init> (Lnet/minecraft/realms/RealmsButton;IIILjava/lang/String;)V
+		ARG 0 button
+		ARG 1 id
+		ARG 2 x
+		ARG 3 y
+		ARG 4 text
+	METHOD <init> (Lnet/minecraft/realms/RealmsButton;IIILjava/lang/String;II)V
+		ARG 0 button
+		ARG 1 id
+		ARG 2 x
+		ARG 3 y
+		ARG 4 text
+		ARG 5 width
+		ARG 6 height
+	METHOD a released (II)V
+		ARG 0 mouseX
+		ARG 1 mouseY
+	METHOD a setText (Ljava/lang/String;)V
+		ARG 0 value
+	METHOD a getTextureId (Z)I
+		ARG 0 pressed
+	METHOD b getWidth ()I
+	METHOD b drawBackground (Lnone/bdr;II)V
+		ARG 0 game
+		ARG 1 mouseX
+		ARG 2 mouseY
+	METHOD b setEnabled (Z)V
+		ARG 0 value
+	METHOD c getId ()I
+	METHOD c isWithinBounds (Lnone/bdr;II)Z
+		ARG 0 game
+		ARG 1 mouseX
+		ARG 2 mouseY
+	METHOD c getTexId (Z)I
```

Comment:
It's called in one place, `getYImage` (not obfuscated) in `net.minecraft.realms.RealmsButton`. Which is only called by `getTextureId` in `WidgetRealmsButton`, so just kind of loops around?

It seems it may be some kind of wrapper object, so `getActualTextureId`?